docs/reference/index.md

```diff
@@ -0,0 +1,69 @@
+# API Reference
+
+Welcome to the TruLens API Reference! Use the search and navigation to explore
+the various modules and classes available in the TruLens library.
+
+## Required and Optional packages
+
+These packages are installed when installing the main `trulens` package.
+
+- `trulens-core` installs [core][trulens.core].
+
+- `trulens-feedback` installs [feedback][trulens.feedback].
+
+- `trulens-dashboard` installs [dashboard][trulens.dashboard].
+
+- `trulens_eval` installs [trulens_eval](trulens_eval), a temporary package for backwards compatibility.
+
+Two categories of optional packages contain integrations with 3rd party app
+types and providers:
+
+- [Apps](apps/index.md).
+
+    - [TruChain][trulens.apps.langchain.TruChain] in package
+        `trulens-app-langchain`.
```

Comment:
nit: trulens-apps-langchain

same for others in this file

Reply:
thx, fixing in another PR